app/src/main/java/org/xbmc/kore/host/HostConnectionObserver.java

```diff
@@ -592,9 +593,29 @@ public void onSuccess(ArrayList<GetPlaylist.GetPlaylistResult> result) {
                 if (result.isEmpty())
                     return;
 
-                for (PlaylistEventsObserver observer : playlistEventsObservers) {
-                    observer.playlistsAvailable(result);
+                boolean isPlaylistChanged = false;
+                for (GetPlaylist.GetPlaylistResult getPlaylistResult : result) {
+                    if (! isPlaylistChanged &&
```

Comment:
The `isPlaylistChange` check here is not necessary (it just avoids another assignment, but if makes reading the code more difficult, as you need to keep another variable in mind, which is burdensome and unnecessary).